src/operators/operators.h

```diff
@@ -103,13 +112,14 @@ class OperatorBase
      *  The tuple is built from create_collection_tuple(). */
     OperatorBase( 
             const Parameters::AllParameters *const parameters_input,
+            const int nstate_input,//number of states input
             const unsigned int degree,
             const unsigned int max_degree_input,
             const unsigned int grid_degree_input,
             const MassiveCollectionTuple collection_tuple);
 
     ///The collection tuple.
-    MassiveCollectionTuple create_collection_tuple(const unsigned int max_degree, const Parameters::AllParameters *const parameters_input) const;
+    MassiveCollectionTuple create_collection_tuple(const unsigned int max_degree, const int nstate, const Parameters::AllParameters *const parameters_input) const;
```

Comment:
Copy pasted from DG.

Reuse functions by moving things around, don't copy paste.

Reply:
fixed

Reply:
How has this been fixed?

Reply:
Sorry, you are correct. I will delete the collection tuple in DG and go through the code replacing all references to dg->fe_collection, quadrature etc with its operators equivalent. Once this is done I will comment "fixed".

Reply:
fixed

Reply:
@dougshidong looks like this has been resolved